pkg/convert/pot.go

```diff
@@ -140,11 +133,18 @@ func msgNodeToMF2(node pot.MessageNode, getMessages func(pot.MessageNode) []stri
 		return nil
 	}
 
+	// look for format flag, e.g. #, python-format, c-format, no-python-format, etc.
+	formatFlagIdx := slices.IndexFunc(node.Flags, func(flag string) bool {
+		return strings.Contains(flag, "-format")
```

Comment:
I don't know the flag's value, but shouldn't we check the suffix?

```suggestion
		return strings.HasSuffix(flag, "-format")
```

Reply:
you are right